go.mod

```diff
@@ -33,3 +34,5 @@ require (
 	gopkg.in/square/go-jose.v2 v2.4.1
 	k8s.io/apimachinery v0.19.3
 )
+
+replace github.com/oauth2-proxy/tools/reference-gen => github.com/oauth2-proxy/tools/reference-gen v0.0.0-20201129112127-38b1e16b3e13
```

Comment:
Just wanted to confirm this was intended to get commited

Reply:
At present, the generator is sitting on a branch, so until that is merged to master, we need a replacement... How do you feel about merging this without merging the other to master?  🤔 Let me get a PR open